documentation/app/templates/index.hbs

```diff
@@ -0,0 +1,96 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+}}
+
+<PageHeader @title="Overview" />
+
+<Hds::Tabs as |T|>
+  <T.Tab>About</T.Tab>
+  <T.Tab>Support</T.Tab>
+  <T.Tab>Contributing</T.Tab>
+  <T.Panel>
+    <p>Consul UI Toolkit is a collection of presentational components and
+      utilities specifically meant to be shared across the different Consul
+      applications. The toolkit contains generic components that you may have
+      come to expect from a component library, as well as very feature specific
+      components meant for the Consul apps</p>
+
+    <p>While the toolkit is an open-source project, it is intentionally built
+      for Consul products. However, it may be used outside of them. Please refer
+      to the
+      <Hds::Link::Inline
+        @href="https://github.com/hashicorp/consul-ui-toolkit/blob/main/LICENSE"
+      >License</Hds::Link::Inline>
+      before using the toolkit in your own projects.</p>
+
+    <h2>Compatibility</h2>
+    <ul>
+      <li>Ember.js v3.28 or above</li>
+      <li>Embroider or ember-auto-import v2</li>
+    </ul>
+
+    <h2>Installation</h2>
+    <CodeBlock @code="ember install @hashicorp/consul-ui-toolkit" />
+  </T.Panel>
+  <T.Panel>
+    <Hds::Alert @type="inline" as |A|>
+      <A.Description>
+        Many resources and links on this page are only available to HashiCorp
+        employees.
+      </A.Description>
+    </Hds::Alert>
+
+    <p>We’re happy to provide support to HashiCorp product teams that use Consul
+      UI Toolkit. Use these resources to get help.</p>
+
+    <h2>Live Support</h2>
+    <p>Join the
+      <Hds::Link::Inline
+        @href="https://hashicorp.slack.com/archives/C9X0VGJNL/p1681481135411859"
+      >#team-consul-ui</Hds::Link::Inline>
+      Slack channel to ask questions, request and provide feedback, or start a
+      conversation with our team. As a globally-distributed team, there may be a
+      slight delay in answering some questions, but we’re committed to
+      responding in a timely manner.</p>
+
+    <h2>File an Issue</h2>
+    <p>Find a bug? Want to report a security issue?
+      <Hds::Link::Inline
+        @href="https://github.com/hashicorp/consul-ui-toolkit/issues/new"
+      >File an issue in our GitHub repository</Hds::Link::Inline></p>
+  </T.Panel>
+  <T.Panel>
+    <Hds::Alert @type="inline" as |A|>
```

Comment:
For the two alerts shown can we update to use the "highlight" (purple) alert type. Can we also use the "title" for content that is a single line item and only use the "description" to show additional secondary info? Shown like the below: 
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/91899352/233190902-e45b1a75-8c18-4e4a-91ab-e439053022ff.png">


<img width="546" alt="image" src="https://user-images.githubusercontent.com/91899352/233190621-2013f7a7-9e04-42dc-a887-22e9001af9f1.png">